lms/migrations/versions/911d1f7da759_backfill_course_groupings.py

```diff
@@ -0,0 +1,70 @@
+"""
+Backfill grouping with courses based on rows from `course` and `group_info`.
+
+Revision ID: 911d1f7da759
+Revises: 2119e1c621de
+Create Date: 2022-03-25 11:35:19.760847
+
+"""
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "911d1f7da759"
+down_revision = "2119e1c621de"
+
+
+def upgrade():
+    conn = op.get_bind()
+
+    conn.execute(
+        """
+        INSERT INTO grouping
+            (
+                created,
+                application_instance_id,
+                authority_provided_id,
+                parent_id,
+                lms_id,
+                lms_name,
+                type,
+                settings,
+                extra
+            )
+        SELECT
+                -- Using a hard-coded value in the past to identify the rows added by the migration
+                '2022-03-25 00:00:00',
+                application_instances.id,
+                group_info.authority_provided_id,
+                NULL,
+                context_id,
+                context_title,
+                'course',
+                course.settings,
+                CASE WHEN
```

Comment:
This should be the same behavior as:

https://github.com/hypothesis/lms/blob/0af036b91e6ad5a6143429c99d5bf90d89b2879d/lms/resources/lti_launch.py#L244